aiodns/__init__.py,sha256=cWGNZI1vCare6bxe12v8bcHbFFT2NBm3TBoa4lMmTzk,4663
aiodns/error.py,sha256=fnNPLc8ceLk9drumwt-6Un5iiowyWApTe_VIoJ5gT5U,134
aiodns-2.0.0b0.dist-info/LICENSE,sha256=6wRV01EpQl7TmYg81xCSPePiRqUQ4uuE2fAAMtC77Jc,1070
aiodns-2.0.0b0.dist-info/METADATA,sha256=T31_6gi9xZsvTqEbdoiF0oV8z45LAwuIK6NPOtm69aw,3324
aiodns-2.0.0b0.dist-info/WHEEL,sha256=_wJFdOYk7i3xxT8ElOkUJvOdOvfNGbR9g-bf6UQT6sU,110
aiodns-2.0.0b0.dist-info/top_level.txt,sha256=5J2m3NWP4dezFZNpQoHaj8mv472iPcUzFboQoqWcEe4,7
aiodns-2.0.0b0.dist-info/RECORD,,
